test/components/plots/piePlotTests.ts

```diff
@@ -164,5 +164,17 @@ describe("Plots", () => {
       });
 
     });
+
+    it("throws warnings on negative data", () => {
+      var message;
+      var oldWarn = Plottable._Util.Methods.warn;
+      Plottable._Util.Methods.warn = (warn) => message = warn;
+      piePlot.removeDataset("simpleDataset");
+      negativeDataset = new Plottable.Dataset([{value: -5, value2: 10, type: "A"}, {value: 15, value2: 10, type: "B"}]);
```

Comment:
I don't think the value2 part is necessary.  Removing this attribute would mke the test simpler